### ISSUE TITLE
Design shrapnel

### ISSUE THREAD

**Issue by edwh** [Freegle/iznik-nuxt#142]
Little bits and pieces which could be slightly prettier, all fairly quick.

- [ ] MyData Style download as a button. We need to have it as a link so that we can use the download attribute.
- [ ] About - use a bootstrap container rather than spacer columns.
- [ ] About - Rework to use the FAQ accordion as per help page
- [ ] Disclaimer - use bootstrap container
- [ ] Terms - use bootstrap container
- [ ] Privacy - use bootstrap container
- [ ] error-toast - needs love
- [ ] DonationThermometer - animate change in value
- [ ] DraggableMap used from Spread - icon pin in the centre is not visible enough on the map.
- [ ] GroupHeader There's a max-height hack above to keep the description from hogging the screen.  It's not that pretty and some people may not even notice the scrollbar, but could be neater.
- [ ] GroupSelect - hard-coded max-width.
- [ ] OutcomeModal - Needs visual push to make people select a user.  How?
- [ ] OutcomeModal - "Please choose" is red but the dropdown shouldn't be.
- [ ] Ratings - Showing your own feedback in blue is not intuitive.  How can we improve that given the very limited space for this?
- [ ] default - Shrink navbar on scroll?  Hardcoded navbar heights in other places, though, search for 68px.
- [ ] default - Notifications dropdown list isn't wide enough before it's loaded.
- [ ] message/_id - when it doesn't exist, the

 message is brutal.  error variant is a very shouty red.
StartEndDate - should be fieldset within a form.

**Comment by aaronferris** [comment]
With respect to the Group header, how do we want to position the buttons (join etc) on mobile?

Both Under the logo? Just contact under the logo?

![Screenshot 2020-01-13 at 09 17 14](https://user-images.githubusercontent.com/4068825/72244186-9f9ef880-35e5-11ea-8520-edb39951434b.png)

**Comment by edwh** [comment]
Generally I don't want to take up too much vertical space.

So at that screen size:
The group logo looks too large and there's loads of whitespace.You could 
have a shrunken logo on the left side. Then on the right have the group 
name and buttons below.Then the tag line and everything else below it at 
full width.The white card could be full width on the screen (no flowery 
border) and have smaller padding round the contents.
I'd rather reposition things to reduce the amount of blank space than 
aim for consistent positioning across different screen sizes.  It's 
quite fiddly.

------ Original Message ------

**Comment by aaronferris** [comment]
So we want **join** and **contact volunteers** to be consistently positioned across devices? IE Join above, then contact colunteers below. With the logo to the left? Whatever size it needs to be in order to accomodate the consistent button positions?

**Comment by aaronferris** [comment]
OK - think I have that right in my head, probably best to mock something up for mobile, tablet, desktop then confirm before we make any changes,

**Comment by edwh** [comment]
Join and Contact should be close together and if one is above the other 
then it should be Join above.

Side by side would be ok too.  For example I can imagine that at some 
sizes you could have:
Logo on leftGroup name and both buttons in a single lineTagline and rest 
below
...whereas if you put one button on top of the other then everything 
might get pushed down a bit so that more vertical space was used.

Probably anything sensible you do will be better than it currently 
behaves- so yes, mock some stuff up and we can discuss.

Thanks for looking at this.

------ Original Message ------

**Comment by aaronferris** [comment]
Yeah ill take a look when I get chance, back at work now so might be a few days before I can, was just having a quick look to define what the expected result would be.

**Comment by JayyajGH** [comment]
I've just removed the "tidy the group header" issue as it was recently resolved.
